### PR TITLE
remove Draft

### LINKED ISSUE #191: Draft Debug: Stopping debug to restart hangs on connection still there
If for some reason, the connection is still there -- in this case, I stopped the draft-up process because I had forgotten to establish the container registry context -- you get the following. 

```
example-nodejs: Releasing Application: SUCCESS ⚓  (1.5145s)
Inspect the logs with `draft logs 01CC3FE3P7MP8QYKJEVVYN6YY4`
Error: could not find an available port: listen tcp :8080: bind: address already in use
```

Error message, at the minimum, should offer help breaking that connection.

### LINKED ISSUE #192: Draft Debug: Enable F5 to work against minikube as well as remote clusters
Currently in 0.1.8, F5 support for Draft Debug works fine with remote registries and clusters, but it doesn't check the current docker context and therefore is unable to pick up and use the `eval $(minikube docker-env)` context with F5. 

Per https://github.com/Azure/draft/blob/master/docs/install-minikube.md#configure-docker, this works in Draft on the command line, but not in VS Code. The build gets pushed to the wrong repository, resulting in error image pull. 

Love to respect the docker context of local, as draft does on in the shell.

### LINKED ISSUE #193: Draft Debug: "Debug adapter process has terminated unexpectedly"
On saving a file having made a change, the draft up process is recycled, but this occurs:

image

Even though the draft debugger goes through the deployment loop. HOWEVER, once this has occurred, you cannot do the iteration dance a second time by making changes. Nothing happens, likely because the first error disconnected the file watching mechanism.

### LINKED ISSUE #246: Add Draft apps to cluster tree view
Probably with an option to hide it for users of other tools who don't want Draft taking up valuable real estate.

### LINKED ISSUE #247: Add an explorer for Helm chart repos (and Draft pack repos?)
These are workspace side not cluster side so not in the cluster tree view,

### LINKED ISSUE #284: Draft up watch not working?
How is the file changes watch for Draft supposed to work?
Can´t get the file watching for the Draft Up to work. Is there any specific config needed?
Using the latest version (0.1.10) of this extension, and the latest released version (0.15.0) of Draft.
Manual draft up in command line and via VSCode works fine.

Language: Same result for both NodeJS and Java.

Expected behaviour: Draft up should re-deploy the application on changes to one of the main application files.

Observed behaviour: nothing happens, must re-deploy by manually re-triggering Draft Up

### PR DESCRIPTION
**Pull request by bacongobbler**
Draft is now an archived project.

closes #191
closes #192
closes #193
closes #246
closes #247
closes #284

### CODE REVIEW

**Review by itowlson — CHANGES_REQUESTED** [review]
Thanks - just one issue where it looks like there was an accidental deletion of docs that are still needed.  Otherwise looks good!

File: README.md [hunk 1]
```diff
@@ -185,44 +182,11 @@ Minikube tools to be installed and available on your PATH.
...
-
-[Draft](https://kubernetes.io/blog/2017/05/draft-kubernetes-container-development/) is a tool to simplify the process of developing a new Kubernetes application, by creating the necessary deployment components and by keeping code in the cluster in sync with the code on your computer.
-
-  * `Draft: Create` - Set up Draft in the current folder (prerequisite for `Draft: Up`)
-  * `Draft: Up` - Runs Draft to package the current folder and push it to your cluster. To allow for cluster changes,
-  * `Draft: Version` - Get the version of the local Draft client
-
-**NOTE:** Draft itself is in 'draft' form and is not yet stable. So the extension support for Draft is strictly experimental - assumptions may break, and commands and behavior may change!
-
-## Extension Settings
```
[the remark this answers is not in the document]
Comment (bacongobbler) [reply README.md:198]: Oops! Yes, that was unintentional. I only wanted to remove the `draft-path` extension setting. Thanks for catching that.

**Review by itowlson — APPROVED** [review]
Looks good - thanks!

**Review by lstocchi — CHANGES_REQUESTED** [review]
It looks good :+1: . I think the only thing missing is the package.json. Shouldn't we remove `draft` settings/properties from that as well? @bacongobbler

**Review by lstocchi — APPROVED** [review]

**Comment by lstocchi** [comment]
Thank you @bacongobbler :rocket:

**Pull request merged**